pom.xml

```diff
@@ -165,13 +172,19 @@
               <includes>
                 <include>**/*IT.java</include>
               </includes>
+              <excludes>
+                <exclude>**/Windows*IT.java</exclude>
+              </excludes>
             </configuration>
           </plugin>
         </plugins>
       </build>
     </profile>
     <profile>
       <id>ITs</id>
```

Comment:
How does this differ from `onlyITs`? Confused, could use a comment at least.

Reply:
`ITs` will run the "non special" ITs as well as regular tests, `onlyITs` will run **only** the "non special ITs".

The non obvious difference is that one (this one) changes global config for `surefire`, and the other adds a new `surefire` `execution`.

I declare things as special that need a specialised hard coded environment in which to run (ie the windows ITs), they will fail on any regular windows machine with docker, the windows ITs need a windows machine with docker that is joined to a domain that has a well known user with a well known password 😨 )

Reply:
now that they are run anyway, I guess we could change it to only have one profile and then let a user set the test with `-Dtest=xxx` or `-Dsurefire.includes` but I thought that @fbelzunc would prefer to keep the way he is used to.

Reply:
5b890d8ae88c666abd8a2a687efd14066fe7abd3